### PR TITLE
Added vidarr labels

### PR DESCRIPTION
**Pull request by pruzanov**

### CODE REVIEW

File: smallBamQc.wdl [hunk 1]
```diff
@@ -122,12 +122,24 @@ workflow smallBamQc {
...
             }
         ]
-        output_meta: {
-            samtools: "Samtools stats output",
-            picard: "Picard MarkDuplicates output",
-            bedtoolsCoverage: "Bedtools coverage histogram output",
-            downsampledCounts: "JSON file recording what downsampling was done"
-        }
+    output_meta: {
+    samtools: {
```
Comment (bibilujan) [inline smallBamQc.wdl:126]: should samtools and picard labels be more descriptive, ie. "samtoolsMetrics" and "picardMarkDuplicateMetrics"? or is that too wordy?

Comment (pruzanov) [reply smallBamQc.wdl:126]: Not sure, it looks fine to me. I am not adding labels, just re-formatting.

**Review by bibilujan — APPROVED** [review]

**Review by slazicoicr — APPROVED** [review]
